test/mocha.opts

```diff
@@ -1,3 +1,3 @@
 --exit
 --timeout 30000
-build/test/**/*.js
+build/test/**/*.spec.js
```

Comment:
Add `.spec` specificity to prevent any other files from running aside from those explicitly required by test suites, as some files might have side-effects immediately upon import.

Reply:
I could've sworn i did this here https://github.com/balena-os/balena-supervisor/pull/1656/files#diff-7256b382c196e793feaa018e2a9d0d7a68fb0ede44d938a5c41663a21275140cL3

Reply:
@pipex  That change looks like it was `build/test/*.js` --> `build/test/**/*.js`, this one specifies further from that with `build/test/**/*.js` --> `build/test/**/*.spec.js`.

Reply:
Yeah, I know, I meant to do on that PR what you did here

Reply:
Then it's fixed! :D